configuration/configuration.go

```diff
@@ -913,6 +913,36 @@ ZzBJ0G5JHPtaub6sEC6/ZWe0F1nJYP2KLop57FxKRt0G2+fxeA0ahpMwa2oMMiQM
 BA/cKaLPqUF+08Tz/9MPBw51UH4GYfppA/x0ktc8998984FeIpfIFX6I2U9yUnoQ
 OCCAgsB8g8yTB4qntAYyfofEoDiseKrngQT5DSdxd51A/jw7B8WyBK8=
 -----END RSA PRIVATE KEY-----`
+
+	DefaultUserAccountPrivateKey = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
There is already DevModeRsaPrivateKey. Why not reusing it?

Reply:
removed!